documentation/Migrate.md

```diff
@@ -0,0 +1,16 @@
+## Migration from Settings.StyleCop to FxCop.ruleset (using F#)
```

Comment:
:thought_balloon: I'm a bit torn on this. It looks like a lot of work for relatively little gain. It's quite easy in my experience to use the rule set editor to configure the severity of rules. I have the following additional specific concerns:

1. This only covers the severity of rules. Users will still have to configure **stylecop.json** separately and/or reconsider the severity of rules in cases where the behavior of this project does not precisely match StyleCop "classic".
1. This script could enable rules which are currently disabled because they are either not implemented or are completely untested. It seems like it could manually configure the severity of rules which are permanently disabled, such as SA1603 (DocumentationMustContainValidXml) (see #1324). When rules are permanently disabled, a diagnostic descriptor for them is retained in order for the status site to extract information from them (it doesn't use it currently), but the diagnostic is hidden from the rule set editor interface.